chain/consensus/hierarchical/subnet/submgr/manager.go

```diff
@@ -455,6 +462,8 @@ func (s *Service) AddSubnet(ctx context.Context, params *hierarchical.SubnetPara
 		return address.Undef, aerr
 	}
 
+	stats.Record(ctx, metrics.SubnetCreatedCount.M(1))
```

Comment:
It's probably better to collect this metric from the state, what if adding the subnet fails and we record as if it was created when it's not really the case?

Reply:
I'll remove all the current cross net stats. I'll do it in the event listener.